cmd/sourced/cmd/sql.go

```diff
@@ -15,7 +18,11 @@ type sqlCmd struct {
 }
 
 func (c *sqlCmd) Execute(args []string) error {
-	command := []string{"exec", "gitbase", "mysql"}
+	command := []string{"exec"}
+	if !(terminal.IsTerminal(int(os.Stdout.Fd())) && terminal.IsTerminal(int(os.Stdin.Fd()))) {
```

Comment:
```suggestion
	if !terminal.IsTerminal(int(os.Stdout.Fd())) || !terminal.IsTerminal(int(os.Stdin.Fd())) {
```

Reply:
I don't mind one way or the other. But I'm wondering, why do you think this one is preferable?

Reply:
I find easier to understand
`!a || !b` vs `!(a && b)`
specially in that long line with many parenthesis

Reply:
done and squashed.